Applications/FileManager/main.cpp

```diff
@@ -37,6 +38,9 @@ int main(int argc, char** argv)
 
     GApplication app(argc, argv);
 
+    Optional<FilePathList> files_to_copy;
```

Comment:
Let's use the system clipboard for this instead.
We can just put a list of path strings on the clipboard.
That way, you get something sensible when you Ctrl+C in FileManager and Ctrl+V in TextEditor :)